avm/res/cognitive-services/account/main.bicep

```diff
@@ -640,3 +659,17 @@ type lockType = {
   @description('Optional. Specify the type of lock.')
   kind: ('CanNotDelete' | 'ReadOnly' | 'None')?
 }?
+
+type deploymentsType = {
+  @description('Optional. Specify the name of cognitive service account deployment.')
+  name: string?
+
+  @description('Required. Properties of Cognitive Services account deployment model.')
+  model: object
```

Comment:
Should model be a custom type?

Reply:
Good idea. Note @zedy-wj UDTs can be be nested, you don't need a seperate type.